### ISSUE TITLE
Bad file descriptor when closing terminal

### ISSUE THREAD

**Issue by rwols** [Wramberg/TerminalView#7]
I keep getting this exception when I close a terminal view:

```python
Traceback (most recent call last):
  File "/Users/rwols/Library/Application Support/Sublime Text 3/Packages/TerminalView/TerminalView.py", line 170, in _check_for_screen_resize
    self._shell.update_screen_size(self._terminal_rows, self._terminal_columns)
  File "/Users/rwols/Library/Application Support/Sublime Text 3/Packages/TerminalView/LinuxPty.py", line 44, in update_screen_size
    fcntl.ioctl(self._pts, TIOCSWINSZ, s)
OSError: [Errno 9] Bad file descriptor
```

**Comment by Wramberg** [comment]
Doesn't happen to me on Linux. Seems like the closing of the view triggers a resize event which we try to send to the shell after its closed. Could you try to see if 7e1ba6e630b9067039e96f8ee0899c0e7f4880e7 fixes it ?

**Comment by rwols** [comment]
The extra if condition in 7e1ba6e seems to do the trick, the exception is gone.

**Comment by rwols** [comment]
I seem to have been too eager to report this, it doesn't always help -- It's probably a race condition somewhere.

**Comment by Wramberg** [comment]
Fixed by 9c6cb2fd4f6189a5f0c097bc49649fd92dab7a9a
